src/main/scala/com/codacy/client/stash/BuildStatus.scala

```diff
@@ -28,7 +28,7 @@ case class TimestampedBuildStatus(
   name: String,
   url: String,
   description: String,
-  dateAdded: String
+  dateAdded: Long
```

Comment:
This should be a DateTime like other endpoints similar to https://github.com/codacy/stash-scala-client/blob/master/src/main/scala/com/codacy/client/stash/Commit.scala#L14

Reply:
Will do.